src/Text/ExcerptHelper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+* Excerpt
+* (c) Matteo Merola
```

Comment:
This license is for the library, not for the single file, so you have to use the same header as used in other files.